hicexplorer/hicBuildMatrix.py

```diff
@@ -14,12 +14,6 @@
 
 debug = 1
 
-# check pysam version
-from distutils.version import LooseVersion
-if LooseVersion(pysam.__version__) < LooseVersion("0.8.3"):
-    exit("\n*ERROR*\n\nVersion of pysam has to be higher than 0.8.3. Current installed version is {}\n".format(pysam.__version__))
-
-
```

Comment:
RTD was showing build issues precisely here (and i thought it was redundant with the setup).

Although the build error still exists :(